aws/aws-sdk-java-v2
# Redirect to correct URL for evenstream operations

Fix issues related to correctly constructing the redirect URL for eventstream operations:

 - Eventstream operations are only available on the async client
 - Eventstream operations have additional method parameters



## Motivation and Context



## Modifications


## Testing




 - Create a directory `javadoc-root`
 - Unzip the [generated javadoc](https://github.com/aws/aws-sdk-java-v2/blob/c6c9d6a20944495e61af684e4d8161392abf37ca/buildspecs/release-javadoc.yml#L26) JAR into `javadoc-root`
 - Copy the generated `crosslink_redirect.hml` file to `javadoc-root`
 - Start a simple Python web server (serves HTTP on port 8000 by default)
    `python -m http.server`
 - Open a browser and nagivate to `http://localhost:8000/crosslink_redirect.html?uid=s3-2006-03-01&type=SelectObjectContent` and verify that it redirects correctly
 - Also tested other non event streaming operations like S3 `ListBuckets`, bidirectional eventstreaming on Transcribe's `StartStreamTranscription`
## Screenshots (if appropriate)

## Types of changes


## Checklist



## License
